terraform/cellxgene.tf.json.template.py

```diff
@@ -150,13 +150,38 @@ def matrix_files():
                 "private_zone": False
             }
         },
-        "aws_iam_role": {
-            "cellxgene": {
-                "name": "ecsTaskExecutionRole"
+        "aws_iam_policy_document": {
+            "ecs-task-assume-role": {
+                "statement": {
+                    "actions": ["sts:AssumeRole"],
+                    "principals": {
+                        "type": "Service",
+                        "identifiers": [
+                            "ecs-tasks.amazonaws.com"
+                        ]
+                    }
+                }
+            }
+        },
+        "aws_iam_policy": {
+            "ecs-task-execution-role-policy": {
+                "arn": "arn:aws:iam::aws:policy/service-role/AmazonECSTaskExecutionRolePolicy"
             }
         }
     },
     "resource": {
+        "aws_iam_role": {
+            "cellxgene": {
+                "name": "cellxgene",
+                "assume_role_policy": "${data.aws_iam_policy_document.ecs-task-assume-role.json}"
```

Comment:
Assume role policies (and other policies that are referenced exactly once) I prefer not to go though the `data` indirection.

https://github.com/DataBiosphere/azul/blob/0964b1c2c4eff3d91b0b21b63af183793c15cb80/terraform/gitlab/gitlab.tf.json.template.py#L837